src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -200,4 +207,24 @@ public function isRaw()
     {
         return $this->raw;
     }
+
+    /**
+     * Gets the SameSite attribute.
+     *
+     * @return string|bool
+     */
+    public function getSameSite()
+    {
+        return $this->sameSite;
+    }
+
+    /**
+     * Checks if the cookie value should be sent with a SameSite attribute.
+     *
+     * @return bool
+     */
+    public function hasSameSite()
```

Comment:
As it's not a pure boolean property, you could also consider `getSameSite` and instead of `false` use `null` as default value.. (so it's more in line with `set/getDomain` for instance, a key-value property).